db-migration.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+BOLD=$(tput bold)
+RED=$(tput setaf 1)
+GREEN=$(tput setaf 2)
+YELLOW=$(tput setaf 3)
+RESET=$(tput sgr0)
+
+## VARS
+OUTPUT=/tmp/github-for-jira.dir
+JOBS=$(($(nproc --all)-2)) # Number of cpu cores minus 2 to do parallel restore jobs
+SOURCE_URL="$1"
+TARGET_URL="$2"
```

Comment:
The URL is just `hostname:port`?

Reply:
It's a DB connection string, like you can find for the `PG_DATABASE_URL` env var.

Reply:
example `postgres://postgres:postgres@127.0.0.1:5432/jira-dev`